src/integration-tests/test_reconfigure_domains.py

```diff
@@ -322,7 +324,7 @@ def do_test(expect_success):
             if expect_success:
                 client.confirm(URI, "+1", succeed=True)
```

Comment:
For this `confirm`, we don't have to call `wait_push_event`, since we expect success from `client.confirm` here. Fflag `succeed=True` means that we wait for some timeout until we confirm something.